ci-operator/config/openshift/csi-operator/openshift-csi-operator-master.yaml

```diff
@@ -7,7 +7,7 @@ binary_build_commands: make build
 build_root:
   from_repository: true
 images:
-- dockerfile_path: Dockerfile.aws-ebs
+- dockerfile_path: Dockerfile.rhel7
```

Comment:
This has broken CSI operator builds. We really want `Dockerfile.aws-ebs` to be used. Where `Dockerfile.rhel7` comes from?